doc/decisions/frontend_end_to_end_testing.md

```diff
@@ -0,0 +1,175 @@
+# E2E Testing
+
+as of 12.07.2023
+
+## Problem
+
+There is currently no existing e2e tests locally and in the pipeline.
+It lacks profound decision making which framework to pick and the frontend team needs to be d'accord with it.
+
+## Constraints
+
+1. [Behaviour-driven development](https://www.selenium.dev/documentation/test_practices/testing_types/#behavior-driven-development-bdd)
+2. [Page Object Design Pattern](https://www.selenium.dev/documentation/test_practices/encouraged/page_object_models/)
+3. Fast and easy prototyping of the tests
+
+## Assumptions
+
+- Understanding about BDD practices.
+- Understanding about the Page Objects Design Pattern.
+- Understanding about end-to-end tests.
```

Comment:
I don't think this is given, we should create (short) guidelines, see #626

Reply:
I don't want to duplicate information that is already given on other websites. Maybe a link each that redirects to a guide would be enough?